django/test/utils.py

```diff
@@ -640,3 +640,14 @@ def setUp(self):
 
     def tearDown(self):
         self.logger.handlers[0].stream = self.old_stream
+
+
+def tag(*tags):
+    """
+    Decorator to add tags to test
+    """
+    def decorator(function):
```

Comment:
I'm not opposed to this is if it's really easy to do, but I also think that

a) it's a bit more magical than the rest of the feature (relies on a magic top-level name in the module).

b) It's not that important -- generally you shouldn't have test modules with so many test cases in them that it would be such a difficult thing to annotate each test case class.

We aren't trying to achieve feature-parity with py.test here, just implement something minimal and useful.